geocoding_android/example/pubspec.yaml

```diff
@@ -25,19 +25,13 @@ dependencies:
   # The following adds the Cupertino Icons font to your application.
   # Use with the CupertinoIcons class for iOS style icons.
   cupertino_icons: ^1.0.2
-  url_launcher: ^6.0.2
+  url_launcher: ^6.0.36
```

Comment:
`url_launcher: 6.0.36` does not exist. Is this a typo?

Reply:
That was indeed a mistake, I mixed the `url_launcher` and `url_launcher_android` packages.